CHANGELOG.md

```diff
@@ -1,5 +1,39 @@
 ## CHANGELOG
 
+### 5.1.0 (TBD)
+**SDK**
+- **Feature:** Universal Bridge support for receiving the Json or string content from the MiniApp to host app. 
+- **Feature:** Added new public protocol `MiniAppBridgeDelegate` which is confirmed by `MiniAppMessageDelegate`, further added public interface `sendJsonToHostApp(info:, completionHandler:) -> Void)` in `MiniAppBridgeDelegate` which can be implemented in host app to receive the string content from mini app and use in host app through Universal Bridge.
+<details>
+<summary>Show code</summary>
+
+```swift
+func sendJsonToHostApp(info: MiniAppBridgeContent, completionHandler: @escaping (Result<MASDKProtocolResponse, Error>) -> Void) {
```

Comment:
'Error' ?

Reply:
This is updated in the later commits. Please review.